pkg/runtime/runner_test.go

```diff
@@ -0,0 +1,59 @@
+package runtime
```

Comment:
Test cases for the runner are missing, would be nice to add some test cases for it. Maybe `runtime_test.go` has something which can be moved in here.

Reply:
Sure, I can just pull a similar test out of `runtime_test.go` and modify it for the `runner`. I was debating on mocking runner.go. But we would get into an interface implementation which isn't needed at the moment. Maybe another good refactor. I'll add this Friday morning.